cristin-person/src/test/java/no/unit/nva/cristin/person/update/UpdateCristinPersonHandlerTest.java

```diff
@@ -54,14 +64,51 @@
 import static nva.commons.apigateway.RestRequestHandler.EMPTY_STRING;
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 public class UpdateCristinPersonHandlerTest {
 
+    public static final String NVI_JSON_AT_185 = """
+        {
+            "preferredFirstName" : "Erik",
+            "nvi": {
+              "verifiedBy": {
+                "id": "https://api.dev.nva.aws.unit.no/cristin/person/12345",
+                "type": "Person",
+                "firstName": "Tor Inge",
+                "lastName": "Kristianslund"
+              },
+              "verifiedAt": {
```

Comment:
Now, the naming futher up makes more sense... 
Can a person have roles from different organizations in the same context?
This is not something we can change, but  an entity (person) is partof X, X is partof Y, seems more logical and can be extended infinitely.

Reply:
For now, verified by ANY institution is good enough. I have also included the top level organization in case someone decides to verify on sub level organization.